docs/modules/objectives/blitz.mdx

```diff
@@ -27,6 +27,14 @@ Blitz maps should include `<respawn auto="true"/>` to prevent players from "hidi
   | `<join-filter>` | Filters when a player can join the match after it has started. | [Filter](/docs/modules/mechanics/filters) | `never` |
 </div>
 
+##### Blitz Attributes
+
+<div className="table-container">
+  | Attribute | Description | Value |
+  |---|---|---|
+  | `scoreboard-filter` | Determines when the objective should be displayed on the scoreboard.<br />**Note:** This filter is team-scoped, it can filter whether each team should show it or not independently. | [Dynamic Filter](/docs/modules/mechanics/filters#dynamic-filters) |
```

Comment:
Current wording makes it seem like it's the viewer team being filtered, and they get a chance to get it shown or not.

That's not the case, it's the team to display that gets filteredd.

Maybe reword `team should show it or not` to `team should show or not`, there's no separate "it" to show, the team is the "it"